### PR TITLE
Avoid "Warning: Constant LARAVEL_START already defined"

### PR DESCRIPTION
**Pull request by GrahamCampbell**

### CODE REVIEW

File: src/laravel/README.md [hunk 1]
```diff
@@ -24,11 +24,16 @@ APP_RUNTIME=Runtime\Laravel\Runtime
...
 
-return function () {
-    define('LARAVEL_START', microtime(true));
-    $app = require_once __DIR__ . '/../bootstrap/app.php';
+return function (): Kernel {
+    static $app;
+
+    if (null === $app) {
+        $app = require dirname(__DIR__).'/bootstrap/app.php';
+    }
```
Comment (azjezz) [inline src/laravel/README.md:36]: when using amphp, react, roadrunner .. etc, this means that the application shares the same state between all requests, is that considered okay?

Comment (GrahamCampbell) [reply src/laravel/README.md:36]: Yes. That is the entire point. :)

Comment (Nyholm) [reply src/laravel/README.md:36]: I think I got the warning when running this front controller with `artisan`.

**Comment by GrahamCampbell** [comment]
Tested this, now.

**Comment by Nyholm** [comment]
Thank you

**Pull request merged**